src/python/pants/backend/docgen/tasks/generate_pants_reference.py

```diff
@@ -48,7 +48,7 @@ def execute(self):
   def _gen_reference(self):
     def get_scope_data(scope):
       ret = []
-      for si in ScopeInfoIterator(self.context.options.known_scope_to_info).iterate([scope]):
+      for si in ScopeInfoIterator(self.context.options.known_scope_to_info).iterate(set([scope])):
```

Comment:
Nit, always prefer using collection literals for performance and readability.

```suggestion
      for si in ScopeInfoIterator(self.context.options.known_scope_to_info).iterate({scope}):
```